.changeset/twelve-islands-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-theme": patch
+---
+
+Fix styling of filter input fields
```

Comment:
```suggestion
Data Grid: Fix styling of filter input fields
```

Reply:
@johnnyomair should we apply this change?